src/vma/dev/ib_ctx_handler.cpp

```diff
@@ -53,29 +54,76 @@
 #define ibch_logfuncall         __log_info_funcall
 
 
-
-ib_ctx_handler::ib_ctx_handler(struct ibv_context* ctx, ts_conversion_mode_t ctx_time_converter_mode) :
+ib_ctx_handler::ib_ctx_handler(struct ibv_context* ctx, ts_conversion_mode_t m_p_ctx_time_converter_mode) :
```

Comment:
`m_p_ctx_time_converter` is what we by convention call member pointers. Being a function parameter it should be called `ctx_time_converter_mode`.

Please see that Jenkins passes as it is and then change all occurrences in this function.